code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -22,6 +22,7 @@
 	disliked_food = GRAIN | DAIRY
 	liked_food = GROSS | MEAT
 	inert_mutation = FIREBREATH
+	var/datum/action/innate/lizlighter/lizlighter
```

Comment:
This is mostly nitpick, but can you move the new var declaration below existing vars being set

Reply:
sure